src/database/config/config.js

```diff
@@ -0,0 +1,38 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+const {
+  POSTGRES_USER,
+  POSTGRES_PASSWORD,
+  POSTGRES_DB,
+  POSTGRES_HOST,
+  POSTGRES_PORT,
+  POSTGRES_DIALECT,
+} = process.env;
+
+module.exports = {
+  development: {
+    username: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PASSWORD,
+    database: process.env.POSTGRES_DB,
+    host: process.env.POSTGRES_HOST,
+    dialect: process.env.POSTGRES_DIALECT,
+    port: process.env.POSTGRES_PORT,
+  },
+  test: {
+    username: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PASSWORD,
+    database: process.env.POSTGRES_DB,
+    host: process.env.POSTGRES_HOST,
+    dialect: process.env.POSTGRES_DIALECT,
+    port: process.env.POSTGRES_PORT,
+  },
+  production: {
+    username: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PASSWORD,
+    database: process.env.POSTGRES_DB,
+    host: process.env.POSTGRES_HOST,
+    dialect: process.env.POSTGRES_DIALECT,
+    port: process.env.POSTGRES_PORT,
+  },
+};
```

Comment:
```suggestion
require('dotenv/config')
module.exports = {
  development: {
    url: process.env.DEV_DATABASE_URL,
    dialect: 'postgres',
  },
  test: {
    url: process.env.TEST_DATABASE_URL,
    dialect: 'postgres',
  },
  production: {
    url: process.env.DATABASE_URL,
    dialect: 'postgres',
  },

```
For simplicity let use this format for the environment variables
### connection string format
`postgresql://username:password@host:port/dbname[?param1=value1&...]`

Reply:
Thanks, the suggestion has applied.